llvm/lib/Target/AMDGPU/SIInstrInfo.h

```diff
@@ -960,6 +960,19 @@ class SIInstrInfo final : public AMDGPUGenInstrInfo {
 
   unsigned getVALUOp(const MachineInstr &MI) const;
 
+  /// Return true if SCC is deinfed and not dead
```

Comment:
typo `defined`

Also this comment and the ones below could be better. I'd say:
```
\returns true if a non-dead definition of SCC is found between the start of the basic block and \p Before 
```
Same thing for `isSCCUsedAfter`:
```
\returns true if an implicit use of SCC is found between \p After and the end of the basic block
```

Reply:
Good idea.